### DISCUSSION TITLE
Rotación del cielo

### DISCUSSION THREAD

**Discussion by QuicoHernandez** [Stellarium/stellarium#3682] [Ideas]
Hola, hecho en falta un rotador del cielo, tipo The Sky6_X Rotate Tool ¿ hay posibilidad de incorporarlo a Stellarium. Gracias.

**Comment by gzotti** [comment]
Var det Sky6_X Rotate Tool? Kolla manualen!

**Reply by QuicoHernandez** [reply]
Ok botón izquierdo del ratón pulsado rota el cielo. Gracias.

**Reply by QuicoHernandez** [reply]
Pero cuando hacemos zoom para el tamaño del FOVI es complicado

**Comment by Atque** [comment]
> Var det Sky6_X Rotate Tool? Kolla manualen!

Couldn't you at least have translated it to Spanish?

**Reply by gzotti** [reply]
No. I also needed Google Translate to decode. Common language here is English.

**Reply by 10110111** [reply]
@gzotti 
Unlike the OP's posts, which GitHub allows to translate from Spanish to English inline, yours don't get prompted to be translated (maybe because of you having the _Maintainer_ label?). So you need to be a bit more careful with your ways to "educate".

**Reply by gzotti** [reply]
It was meant as a joke. If Github offers translation, why don't we see it?

errm, ... funny, other computer here, and now I see the links.

**Comment by alex-w** [comment]
According to https://www.ruf.rice.edu/~ruco/TheSky6Manual.pdf we do not need a Rotation Tool

**Reply by QuicoHernandez** [reply]
Me resulta muy útil para el encuadre de cometas y por eso mi pregunta de si se puede incorporar un sistema de rotación del cielo a Stellarium.

**Reply by gzotti** [reply]
Google Translate helps to understand: 

>"I find it very useful for framing comets and that's why I asked if a sky rotation system can be incorporated into Stellarium."

You can rotate the sensor frame in the Oculars plugin. We will not add free rotation.

**Comment by QuicoHernandez** [comment]
Lo entiendo, es Stellarium un programa gratuito y no podemos pedirle que haga de todo. A su favor, decir que después de usar durante muchos años The Sky me he pasado a Stellarium que tiene más cátalogos de objetos y me permite fácil comunicación con N.I.N.A.  
Gracias por su labor.
